fleet/view/js/fleet.js

```diff
@@ -0,0 +1,316 @@
+var loc = window.location.pathname;
+var split_url = loc.split('/');
+var url_host = window.location.origin+"/"+split_url[1]+"/"+split_url[2];
+
+var loc_host = window.location.origin
+var split_host = loc_host.split('/');
+
+var id = 0;
+
+if (split_host[2] != 'localhost') {
+    if (window.location.pathname.split('/')[3]) {
+        id = window.location.pathname.split('/')[3];
+    }
+} else {
+    if (window.location.pathname.split('/')[4]) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400071)
```suggestion
    if (window.location.pathname.split("/")[4]) {
```